File: MyMovies.Web/src/models/Movie.js
export default class Movie {
  id = "";
  title = "";
  releaseDate = null;

  constructor(data) {
    if (data) {
      this.id = data.id ? data.id : "";
      this.title = data.title ? data.title : "";
      this.releaseDate = data.releaseDate ? data.releaseDate : "";
    }
  }
}

File: MyMovies.Server/MyMovies.Entities/Entities/Movie.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;

namespace MyMovies.Models.Entities
{
    public class Movie
    {
        [Key]
        public int Id { get; set; }

        [Required, MaxLength(200)]
        public string Title { get; set; }

        [Range(1900, 2100)]
        public int? ReleaseDate { get; set; }
    }
}
